content/docs/command-reference/run.md

```diff
@@ -193,8 +193,8 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
   on, from a parameters file. This is done by sending a comma separated list as
   argument, e.g. `-p learning_rate,epochs`. The default parameters file name is
   `params.yaml`, but this can be redefined with a prefix in the argument sent to
-  this option, e.g. `-p parse_params.yaml:threshold`. See `dvc params` to learn
-  more about parameters.
+  this option, e.g. `-p parse_params.yaml:threshold`. See
+  [parameters](/doc/user-guide/concepts/parameters) to learn more.
```

Comment:
> Right, sometimes an auto-link to the cmd ref may be most appropriate.

Reply:
But actually in this case the default params file name info has been moved to the concepts page so a link to concepts page would be better here... (for now, not sure where I'll keep all that detailed params behavior info).

Reply:
And there's another 2 instances of this change down this dic.

Reply:
doc

Reply:
Replaced these three with tooltips, plus another one in the description. (There are also similar references for Metrics and Plots that may need similar treatment.)